app/views/users/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>List of all users</h1>
+
+<% @users.each do |user| %>
+  Name: <%= user.name %><br>
+<% end %>
```

Comment:
- [ ] Your page shows the error below. Please ensure that all page runs without errors.

<img width="898" alt="image" src="https://user-images.githubusercontent.com/77190203/178373154-a51bc643-2440-4702-990c-c223372d7a4b.png">